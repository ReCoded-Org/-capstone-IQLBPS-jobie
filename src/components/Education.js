import React ,{ useState } from 'react'

const Education = () => {
  const [isShowForm ,setIsShowForm] =useState(false)
  function handleShowForm (){
    setIsShowForm(true )
  }
  function handleRemoveShowForm (){
    setIsShowForm(false)
  }
  return (
    <div  className='bg-blue-200 '>  
             <div className='relative h-32 w-32'>
             <h1>Education</h1>
          <button type="button" 
          onClick={handleShowForm}
           className="bg-blue-600 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-full  left-10 top-10 "
          
          >Add Item</button>
             </div>
  
           {isShowForm &&   
             <div>
                   <span>Education Item</span>
                   <button 
                   type="button" 
                   onClick={handleRemoveShowForm}
                   className="bg-transparent hover:bg-blue-500 text-blue-700 font-semibold hover:text-white py-2 px-4 border border-blue-500 hover:border-transparent rounded-full"
                   >Remove</button>
                 <div className='bg-blue-200'>
                 <form className='grid grid-cols-3 divide-x '>
          <div>
               <lable> University/school</lable>
               <input type="text"  required/>
          </div>
          <div>
               <lable> Major </lable>
               <input type="text"  required/>
          </div>
          <div>
               <lable> Date </lable>
               <input type="date" required/>
          </div>


        
            
          
        </form>
            
                 </div>
             </div>
           
           }
       
    </div>
  )
}

export default Education