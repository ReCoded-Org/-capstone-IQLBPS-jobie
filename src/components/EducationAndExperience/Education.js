import React ,{ useState } from 'react'

const Education = () => {
  const [isShowForm ,setIsShowForm] =useState(false)
  function handleShowForm (){
    setIsShowForm(true )
  }
  function handleRemoveShowForm (){
    setIsShowForm(false)
  }
  return (
    <div  className='bg-[#d5d6d8]'>  
             <div className='pt-4 pb-10'>
             <h1 className="pb-5">Education</h1>
          <button type="button" 
          onClick={handleShowForm}
           className="bg-[#154360]  hover:bg-[#226086] text-white font-bold py-2 px-4 rounded-full  left-10 top-10 "
          
          >Add Item</button>
             </div>
  
           {isShowForm &&   
             <div className='pt-4'>
                   <h1 className='pb-5'>Education Item</h1>
                   <button 
                   type="button" 
                   onClick={handleRemoveShowForm}
                   className="bg-transparent hover:bg-[#154360] text-[#154360] font-semibold hover:text-white py-2 px-4 border border-blue-500 hover:border-transparent rounded-full"
                   >Remove</button>
                 <div className="bg-[#d5d6d8]  flex flex-col justify-center">
                 <form className="max-w-[400px] w-full mx-auto rounded-lg bg-[#d5d6d8] p-8 px-8">
          <div className='flex flex-col text-black-400 py-2'>
               <lable> University/school</lable>
               <input type="text"  required    className='rounded-lg bg-white mt-2 p-2 focus:border-blue-500  focus:outline-none'/>
          </div>
          <div className='flex flex-col text-black-400 py-2'>
               <lable> Major </lable>
               <input type="text"  required   className='rounded-lg bg-white mt-2 p-2 focus:border-blue-500  focus:outline-none'/>
          </div>
          <div className='flex flex-col text-black-400 py-2'>
               <lable> Date </lable>
               <input type="date" required   className='rounded-lg bg-white mt-2 p-2 focus:border-blue-500  focus:outline-none'/>
          </div>


        
            
          
        </form>
            
                 </div>
             </div>
           
           }
       
    </div>
  )
}

export default Education