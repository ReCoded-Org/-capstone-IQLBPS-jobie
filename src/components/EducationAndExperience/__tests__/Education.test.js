import renderer from "react-test-renderer";
import Education from "../Education";
import React from "react";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key) => key }),
  Trans: ({ children }) => children,
}));
it("renders Education commponent correctly", () => {
  const tree = renderer.create(<Education />).toJSON();
  expect(tree).toMatchSnapshot();
});
