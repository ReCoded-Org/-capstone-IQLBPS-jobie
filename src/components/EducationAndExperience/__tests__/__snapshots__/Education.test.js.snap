// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Education commponent correctly 1`] = `
<div
  className="bg-[#d5d6d8]"
>
  <div
    className="pt-4 pb-10"
  >
    <h1
      className="pb-5"
    >
      Education
    </h1>
    <button
      className="bg-[#154360]  hover:bg-[#226086] text-white font-bold py-2 px-4 rounded-full  left-10 top-10 "
      onClick={[Function]}
      type="button"
    >
      Add Item
    </button>
  </div>
</div>
`;
